emailfiling.php

```diff
@@ -155,3 +155,26 @@ function emailfiling_civicrm_alterMailParams(&$params, $context) {
     $hook->run($params, $context);
   }
 }
+
+/**
+ * Implements hook_civicrm_buildForm().
+ */
+function emailfiling_civicrm_buildForm($formName, &$form) {
+  $hooks = [
+    new CRM_Emailfiling_Hook_BuildForm_Setting(),
```

Comment:
Can we make the class name more descriptive 
e.g `CRM_Emailfiling_Hook_BuildForm_AddEmailEmlSetting` or something better. The advantage of making it descriptive is by looking at a glance we are able to know what the hook class is doing without looking into the code.